modules/swagger-codegen/src/test/resources/2_0/petstore.json

```diff
@@ -917,6 +917,10 @@
             "pending",
             "sold"
           ]
+        },
+        "profilePhoto": {
```

Comment:
This will impact all generators. So all examples shall be regenerated

Reply:
We want to keep petstore.json, petstore.yaml the same as the Petstore server (https://petstore.swagger.io) so please undo the change in these files.

I noticed that you already added the additional test case in petstore-with-fake-endpoints-models-for-testing.yaml (there's an similar test case here: https://github.com/benrobot/swagger-codegen/blob/53018c475ad6bbc097f29d47b11a2bc55b5142ea/modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml#L1202)